index.md

```diff
@@ -1,33 +1,34 @@
 *A [Prototype Fund](https://prototypefund.de/project/emissions-api/) project.*
 
 > **tl;dr** The European Space Agency's Sentinel-5P satellite is built to
-> observe emissions (carbon hydroxide, sulfur monoxide, ozone, …). All data
+> monitor air quality data (carbon hydroxide, sulfur monoxide, ozone, …). All data
 > gathered are publicly available …if you know what to do with those data sets,
 > great, but if not:
 >
 > Emissions API's mission is to provide easy access to this data without the
 > need of being an expert in satellite data analysis and without having to
 > process terabytes of data.
 >
-> This way, we hope to empower others to easily build apps like visually showing
-> emissions of countries over time.
+> This way, we hope to empower others to easily build apps that use this data - 
```

Comment:
Use an en-dash with spacing (` – `) or an em-dash without spacing (`—`) instead of a hyphen for something like this:
```suggestion
> This way, we hope to empower others to easily build apps that use this data –
```

Reply:
Fair enough